r/DESCRIPTION

```diff
@@ -7,7 +7,7 @@ Authors@R: c(
     person("Apache Arrow", email = "dev@arrow.apache.org", role = c("aut", "cph"))
   )
 Description: R Integration to 'Apache' 'Arrow'.
-Depends: R (>= 3.5)
+Depends: R (>= 3.1)
```

Comment:
This was a way to eventually be able to use ALTREP without resorting to conditional compilation. If we support older versions of R, this needs to be tested too. Can you also amend the travis config ?

Reply:
Yes for sure, good point. Well, definitely if `ALTREP` ends up working out as expected, etc. we can reconsider, but at least for now, it would help to lower the version. To give you more background, a bunch of scripts in Spark rely on older versions of R, while it's possible to upgrade those clusters, it adds overhead that would be nice to avoid at least while we can.

Reply:
Updated Travis. BTW. I've been testing `arrow` with R 3.1 in an EMR cluster, so far so good, but also hoping all the travis tests succeed 🤞.

Reply:
As a start, looks like we need to add a `GITHUB_PAT` env-variable to the travis settings, @romainfrancois do you have access to Travis? Otherwise, will have to ping Wes.

Reply:
We'd have to ask ASF Infra to do it

Reply:
@javierluraschi I can add it to travis as does @wesm , we don't need INFRA for that.